src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -584,6 +584,16 @@ int getMaxRankingsAllowed() {
         : getNumDeclaredCandidates();
   }
 
+  // function: getMaxSkippedRanksAllowed
+  // purpose: getter for maxSkippedRanksAllowed rule
+  // returns: max skipped ranks allowed in this config
+  // a null value here means "unlimited"
+  Integer getMaxSkippedRanksAllowed() {
+    return rawConfig.rules.maxSkippedRanksAllowed != null
```

Comment:
I don't understand this change. The old logic (just `return rawConfig.rules.maxSkippedRanksAllowed`) behaves identically.

Reply:
you are correct [face palm]